src/scratch/UCERF3/FaultSystemRupSet.java

```diff
@@ -54,7 +54,7 @@
 	
 	// data arrays/lists
 	private List<? extends FaultSection> faultSectionData;
-	private double[] mags;
+	protected double[] mags;
```

Comment:
should be able to just use getter/setters

Reply:
Actually, I'm reverting this file to original now.